packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -233,7 +233,11 @@ export class PendingStateManager implements IDisposable {
 
             // then we push onto pendingStates which will cause PendingStateManager to resubmit when we connect
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            this.pendingStates.push(this.initialStates.shift()!);
+            const message = this.initialStates.shift()!;
```

Comment:
same for this

Reply:
Same as above - these are bugs that existed in the code and my changes uncovered them - exists as stand-alone PR https://github.com/microsoft/FluidFramework/pull/11823 that should be merged before this change